internal/webhook/stage/webhook.go

```diff
@@ -42,23 +42,9 @@ func SetupWebhookWithManager(mgr ctrl.Manager) error {
 }
 
 func (w *webhook) Default(_ context.Context, obj runtime.Object) error {
-	stage := obj.(*kargoapi.Stage) // nolint: forcetypeassert
+	_ = obj.(*kargoapi.Stage) // nolint: forcetypeassert
```

Comment:
What is the function of this line?

Also glad to see the comment about nil checks remains even though this function is basically a no-op now. It will help avoid problems in the future if we have to touch this function again. 👍

Reply:
Not sure why I left it, tbh. I'll remove it.